subprojects/language-native/src/main/java/org/gradle/language/plugins/NativeBasePlugin.java

```diff
@@ -416,4 +419,19 @@ public void execute(MultipleCandidatesDetails<Linkage> details) {
             }
         }
     }
+
+    /**
+     * Used by all native plugins to work around the missing default feature on Property
+     *
+     * See https://github.com/gradle/gradle-native/issues/918
+     *
+     * @since 5.1
+     */
+    public static Set<TargetMachine> setDefaultAndGetTargetMachineValues(SetProperty<TargetMachine> targetMachines, TargetMachineFactory targetMachineFactory) {
```

Comment:
Can we move this to an internal utility class for now?  We don't want this part of the public API

Reply:
We should definitely move this off the public API. Instead of a utility class, it should be something that a base plugin does, instead of having a bunch of different plugins having to remember to do this.